fe/src/utils/GraphQL/ClientNoComponent.re

```diff
@@ -0,0 +1,19 @@
+open Js.Promise;
+
+module Make = (Config: ReasonApolloTypes.Config) => {
```

Comment:
Ten moduł w końcu mi się nie przydał - służy do puszczania requestów graphqlowych bez używania komponentu.
Ale może się w przyszłości okazać przydatny dlatego go narazie nie skasowałem

Reply:
Reaon-apollo dostarcza komponent który na zamontowanie sam pobiera dane. Później można wymusić na nim refetch - zorobiłem wyżej za pomocą tego refetchUsersList - i w zasadzie tyle on potrafi. A czasem mogą być sytuacje że można trzeba jakoś bardziej customowo pobierać dane - np. nie na zamontowanie ale na jakąś akcję. Albo, jeśli chcielibyśmy trzymać dane w stanie komponentu, to wtedy właśnie można użyć tego. To pozwala ręcznie strzelać do be i to coś zwraca promisę. Narazie bym zostawił do momentu zakończenia przepisywania na gql, jeśli po tym będzie niepotrzebny to będzei można wywalić